### ISSUE TITLE
Toggle Tickets & Forums off (settings.yml)

### ISSUE THREAD

**Issue by brianjking** [helpyio/helpy#301]
Hi,

I see that `settings.yml` has the following feature toggles included in it. I set tickets & forums to be false and deployed to Heroku and these features are still available.

Based on this I'm guessing it's not yet fully implemented. When I checked the roadmap on Trello, I couldn't find any mention of when this is planned for.

Please advise. Thanks!

````
# Toggle functionality (not fully implemented)
tickets: true
forums: true
knowledgebase: true
````
https://github.com/helpyio/helpy/blob/master/config/settings.yml#L4-L7

**Comment by brianjking** [comment]
@scott - Sorry to bug you, just wanted to see if you could quickly enlighten me here? Thanks for all you do. :+1:

**Comment by scott** [comment]
Hey @brianjking I actually replied to your comment over here [http://support.helpy.io/en/knowledgebase/18-configuring-your-helpy/docs/22-configuring-your-helpy-settings](http://support.helpy.io/en/knowledgebase/18-configuring-your-helpy/docs/22-configuring-your-helpy-settings) but the bottom line is that those capabilities are not enabled yet.

**Comment by brianjking** [comment]
@scott Thanks, sorry, didn't see that comment. Is this feature on the roadmap anywhere so that I can track it?

**Comment by scott** [comment]
Are you hoping to use it?

**Comment by brianjking** [comment]
Yeah, I really just want to use the knowledge base feature if possible.

**Comment by brianjking** [comment]
@scott - Anyway I can help test this or anything else I can do? Thanks!

**Comment by hadifarnoud** [comment]
I would love this feature too. we prefer to turn forum off.

**Comment by scott** [comment]
Okay, it seems like a pretty common request!

**Comment by brianjking** [comment]
@scott Thank you! I have a question though, even though I logged in and disabled the functionality outside of the knowledgebase there's still a "sign up" option.

What are my options for disabling the sign-up option for end users?

As a follow-up what do I need to enable settings wise to get an email/push notification/admin notification/smoke signal for when people click "yes/no" to an article being helpful?

**Comment by scott** [comment]
Yeah I should probably remove the signup option.  Good catch... The yes/no will automatically fire an event into Google Analytics if you have provided that ID.  Then you can use a conversion event to quickly see the % approval rate for each article.

**Comment by brianjking** [comment]
@scott I've created a new request to track the sign-up disable option in #334.

Thanks for getting this issue closed (#301).
